Samples/Client/DirectxUWP/DirectxClientComponent/AppCallbacks.cpp

```diff
@@ -12,7 +12,7 @@ using namespace Windows::Perception;
 using namespace Windows::Perception::Spatial;
 using namespace Windows::System::Threading;
 
-#define VIDEO_FRAME_WIDTH	1280 * 2
+#define VIDEO_FRAME_WIDTH	1280
```

Comment:
@bengreenier Do you want to set this value by default?

Reply:
probably not, good catch! will revert